File: src/main/java/onl/tesseract/command/MenuCommand.java
package onl.tesseract.command;

import onl.tesseract.commandBuilder.CommandContext;
import onl.tesseract.commandBuilder.annotation.Command;
import onl.tesseract.commandBuilder.annotation.CommandBody;
import onl.tesseract.menu.MenuMenu;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;
import org.jetbrains.annotations.NotNull;

@Command
public class MenuCommand extends CommandContext {
    @CommandBody
    public boolean onCommand(CommandSender sender)
    {
        if (sender instanceof Player player )
        {
            new MenuMenu().open(player);
        }

        return true;
    }
}

File: src/main/java/onl/tesseract/command/MenuCommand.kt
package onl.tesseract.command

import onl.tesseract.commandBuilder.CommandContext
import onl.tesseract.commandBuilder.annotation.Command
import onl.tesseract.commandBuilder.annotation.CommandBody
import onl.tesseract.menu.MenuMenu
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

@Command
class MenuCommand : CommandContext() {
    @CommandBody
    fun onCommand(sender: Player): Boolean {
        MenuMenu(sender).open(sender)
        return true
    }
}
